src/server/routes/main.js

```diff
@@ -4,7 +4,7 @@ const router = require('koa-router')();
 
 router.get('/', async (ctx) => {
   ctx.state.hello = 'world';
-  await ctx.render('index');
+  await ctx.render('index', { hello: 'world' });
```

Comment:
was this for testing?

Reply:
I'll remove that. The "main" router isn't even used anywhere